core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1421,8 +1435,18 @@ public void onReceive(Context context, Intent intent) {
                 PlayerStatus playerStatus = mediaPlayer.getPlayerStatus();
                 if (playerStatus == PlayerStatus.PAUSED || playerStatus == PlayerStatus.PREPARED) {
                     mediaPlayer.resume();
+                    if (PlaybackPreferences.getUserManuallyPaused()) {
+                        mediaPlayer.pause(true, true);
+                    }
                 } else if (playerStatus == PlayerStatus.PREPARING) {
                     mediaPlayer.setStartWhenPrepared(!mediaPlayer.isStartWhenPrepared());
+                } else if (playerStatus == PlayerStatus.STOPPED) {
+                    startPlayingFromPreferences();
```

Comment:
This doesn't feel right. Playing when the status is `stopped`? There must be something else that is going wrong